### PR TITLE
fix: incorrect arguments and including launch

### PR DESCRIPTION
**Pull request by yukkysaito**
Signed-off-by: Yukihiro Saito <yukky.saito@gmail.com>

## Description
Fixed incorrect arguments and including launch.
The logging simulator does not launch the vehicle interface since the vehicle interface often requires the connection of a real machine.

~In other words, the logging simulator is out of the scope of the vehicle interface test.
If you want to test it, you can do so by setting this to true, but the default is false. This will be considered depending on future requests.~

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].
- [x] The PR has been properly tested.
- [x] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: autoware_launch/launch/logging_simulator.launch.xml
Comment (kenji-miyake) [inline autoware_launch/launch/logging_simulator.launch.xml:47]: I think you have to make this controllable by an arg for durability tests.

Comment (yukkysaito) [reply autoware_launch/launch/logging_simulator.launch.xml]: [40c34d8](https://github.com/autowarefoundation/autoware_launch/pull/19/commits/40c34d8014cfed2c53887bbfac3012f8d34fe75b)

**Review by kenji-miyake — DISMISSED** [review]

**Comment by yukkysaito** [comment]
@mitsudome-r Who can merge this PR? :pray:

**Comment by kenji-miyake** [comment]
@yukkysaito These are the permission settings.
image

I think you can be a member of `autoware-maintainers` or `autoware-admins`.

**Comment by yukkysaito** [comment]
@kenji-miyake thank you :pray:

**Comment by yukkysaito** [comment]
sorry, i have a misstake. just a minutes

**Comment by yukkysaito** [comment]
Please wait a little while until I can confirm that it works.

**Comment by kenji-miyake** [comment]
@yukkysaito Fatih-san has added write access to this repository. So please merge this after you confirm it. :+1:

**Comment by yukkysaito** [comment]
> sorry, i have a misstake. just a minutes

fixed at https://github.com/autowarefoundation/autoware_launch/pull/19/commits/5469426e816912f7d1135e74945ad51b12af2684

**Review by kenji-miyake — APPROVED** [review]

**Comment by yukkysaito** [comment]
I confirmed to work following tutorials. 
- `ros2 launch autoware_launch logging_simulator.launch.xml map_path:=/sample/map/ vehicle_model:=sample_vehicle sensor_model:=sample_sensor_kit`
- `ros2 launch autoware_launch planning_simulator.launch.xml map_path:=/sample/map/ vehicle_model:=sample_vehicle sensor_model:=sample_sensor_kit`

This is not related to this PR, but errors from system monitor needs to be fixed.

**Pull request merged**
